File: remote/src/main/java/com/kelsos/mbrc/annotations/Mute.java
package com.kelsos.mbrc.annotations;

import android.support.annotation.IntDef;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;

public class Mute {
  public static final int UNDEF = -1;
  public static final int OFF = 0;
  public static final int ON = 1;

  @IntDef({
      ON,
      OFF,
      UNDEF
  })
  @Retention(RetentionPolicy.SOURCE)
  public @interface State {

  }
}

File: app/src/main/kotlin/com/kelsos/mbrc/annotations/Mute.kt
package com.kelsos.mbrc.annotations

import android.support.annotation.IntDef

object Mute {
  const val UNDEF = -1
  const val OFF = 0
  const val ON = 1

  @IntDef(ON.toLong(), OFF.toLong(), UNDEF.toLong())
  @Retention(AnnotationRetention.SOURCE)
  annotation class State
}
